test/persistence-hooks.suite.js

```diff
@@ -3087,7 +3087,7 @@ module.exports = function(dataSource, should, connectorCapabilities) {
                },
              });
              if (!dataSource.connector.upsertWithWhere) {
-               expectedContext.currentInstance = existingInstance;
+               expectedContext.currentInstance = { id: existingInstance.id, name: 'first', extra: null };
```

Comment:
It is not clear to me why this change is necessary. What is the different between `existingInstance` and plain-data object above? Could you please add a comment explaining this to future readers?